wrappers/python/pysetup.cmake.in

```diff
@@ -1,6 +1,5 @@
 set(ENV{OPENMM_INCLUDE_PATH} "@WRAPPER_INCLUDE_DIRS@")
-file(TO_NATIVE_PATH "@CMAKE_INSTALL_PREFIX@/lib" OPENMM_LIB_PATH)
```

Comment:
Python does *not* like this at all, and as a result `make PythonInstall` would fail if the `CMAKE_INSTALL_PREFIX` had any spaces in the path.

The problem is that distutils essentially does this already.  It takes file paths (both library and include) as a collection of strings.  Each string is a single path, and the way the compilers are invoked internally it ensures that paths with spaces are handled correctly.

What `TO_NATIVE_PATH` does, at least on Linux, is replace spaces with "\ " to escape them.  This is needed when running a shell command with the argument, but breaks it when setting it as an environment variable that Python then interprets as a directory.

Also note that `OPENMM_INCLUDE_PATH` -- which should really be treated the same way -- is set as a raw string without trying to escape any spaces in the path.  That's why the logs show that everything worked... up until linking.  Because libs were only used at linking stage (and the include paths were properly left alone).

This was the trick that was needed to get OpenMM to build on my machine when the install path had a space in it.  Took me forever.